packaging/makeself/build-static.sh

```diff
@@ -21,7 +22,7 @@ esac
 
 DOCKER_CONTAINER_NAME="netdata-package-${BUILDARCH}-static-alpine314"
 
-if [ "${BUILDARCH}" != "$(uname -m)" ]; then
+if [ "${BUILDARCH}" != "$(uname -m)" ] && [ "$(uname -m)" = 'x86_64' ] && [ -z "${SKIP_EMULATION}" ]; then
```

Comment:
Where/When do you plan on using `SKIP_EMULATION`? `grep`-ing the repo does not match anything.

Reply:
For doing local builds on systems where you have QEMU set up correctly natively for binary emulation (this is actually dead simple to set up on Gentoo for example). IOW, it’s something intended to let local builds run a bit faster on certain systems.